src/pages/index.js

```diff
@@ -43,6 +44,24 @@ const simple_step_content = [
     },
 ]
 const Home = () => {
+    /* redirect livechat for en to open live chat popup */
+    let script_timeout = null
+    const [is_mounted, setMounted] = useState(false)
+
+    useEffect(() => {
+        if (is_mounted) {
+            redirectOpenLiveChatBox()
+        }
+
+        script_timeout = setTimeout(() => {
+            setMounted(true)
+        }, 2000)
+
+        return () => {
+            clearTimeout(script_timeout)
+        }
+    }, [is_mounted])
```

Comment:
```suggestion
    /* redirect livechat for en to open live chat popup */
    let script_timeout = null
    const [is_mounted, setMounted] = useState(false)

    useEffect(() => {
        setMounted(true)
        
        if (is_mounted) {
            script_timeout = setTimeout(() => {
                redirectOpenLiveChatBox()
            }, 2000)   
        }

        return () => {
            clearTimeout(script_timeout)
        }
    }, [is_mounted])
```

I think this is cleaner as it wont re-render the page again after 2 seconds. This will instead delay the execution of redirect function for 2 seconds after mounting.

Side note: I think this can be a hook

Reply:
Yep let's try this 👍

Reply:
its working guys..amended in latest commit..